plugins/community/neoeinstein-prost/v0.3.1/buf.plugin.yaml

```diff
@@ -7,3 +7,24 @@ output_languages:
   - rust
 spdx_license_id: Apache-2.0
 license_url: https://github.com/neoeinstein/protoc-gen-prost/blob/protoc-gen-prost-v0.3.1/LICENSE
+registry:
+  cargo:
+    # https://github.com/neoeinstein/protoc-gen-prost/blob/protoc-gen-prost-v0.3.1/protoc-gen-prost/Cargo.toml#L21
+    # https://github.com/tokio-rs/prost/blob/v0.12.3/prost-build/Cargo.toml#L15
+    # TODO: Should we actually set this? This is just the rust-version set by
+    # prost-build, which isn't necessarily the rust-version that the *generated
+    # code* requires.
+    rust_version: "1.60"
+    deps:
+      # https://github.com/tokio-rs/prost/blob/v0.12.3/prost-build/Cargo.toml#L30
+      # TODO: What features are required by the generated code?
+      - name: "prost"
+        req: "0.12.3"
+        default_features: false
+      # prost-types is necessary for any module using the WKTs.
+      - name: "prost-types"
+        req: "0.12.3"
+        default_features: false
```

Comment:
`prost-build` will generate code that depends on the `prost` crate, and any code using the WKT (at least, with the default options set), will depend on `prost-types`.

Another option is [setting the `compile_well_known_types` option](https://github.com/neoeinstein/protoc-gen-prost/tree/protoc-gen-prost-v0.3.1/protoc-gen-prost#options), which would allow us to drop the `prost-types` dependency, but we initially decided on doing the former; I think we should stick with that for now.

The only other concern I have here is the version requirements: we're currently specifying `0.12.3`, which in [Rust-parlance actually means a range of versions](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio) — translating, this means this package supports `0.12.3  :=  >=0.12.3, <0.13.0`. I _guess_ that's fine, but given that the *plugin* is released separately from the underlying generator, I'm not sure if we'd run into situations where e.g. `prost@0.13.0` is released, and users want to upgrade to that version, but the plugin lags in release and thus users aren't able to upgrade.

(Again, I think this is fine for now — mostly leaving notes for the future.)